### PR TITLE
Add Dockerfile with compose script and entrypoint

### PR DESCRIPTION
**Pull request by pjeanjean**
This PR contains:
  - a Dockerfile to create a docker image for the lastest released version of dakara-server
  - an entrypoint to create the database, do a basic configuration (migrations, superuser and tags/wt creation) and run the server after a feed
  - a docker-compose file to set the server port and the path to the kara base

### CODE REVIEW

File: docker/Dockerfile
Comment (Neraste) [inline docker/Dockerfile]: Please concatenate all `apt-get` commands in one line, as explained in the APT-GET subsection of [Dockerfiles best practices documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). You can have cache issues otherwise. Optionally, you can clear the apt cache after.

```dockerfile
RUN apt-get update && apt-get install -y \
    ffmpeg \
    mediainfo \
  && rm -rf /var/lib/apt/lists/*
```

By the way, you do not need mediainfo if you use ffmpeg for extracting metadata.

File: docker/Dockerfile
Comment (Neraste) [inline docker/Dockerfile]: Please put commands to get the server release in a single `get_dakara.sh` script.

File: docker/Dockerfile
Comment (Neraste) [inline docker/Dockerfile]: Just as a remark: the next release has a better settings management system.

File: docker/Dockerfile
Comment (Neraste) [inline docker/Dockerfile]: Tip: when `sed`ing paths, you can use another separator character, to avoid escaping it: `"s+/path/to/file+/path/to/another/file+"`

Comment (Neraste) [inline docker/Dockerfile]: Maybe it is better to use environment variables directly in the `feed.sh` script than `sed`ing it on the way.

File: docker/entrypoint.sh
Comment (Neraste) [inline docker/entrypoint.sh:10]: You need to create the player.

**Review by Neraste — CHANGES_REQUESTED** [review]
Thanks you a lot for your deployment process using Docker.

My review mainly focuses on the Dockerfile, as I spotted several points that should be improved. You should also update the documentation to explain how to use the Docker image.

The next version of Dakara, to be released soon (next weekend), makes the settings management easier. So maybe you should wait a bit to include the new workflow in your PR. Or you can do the adaptation in a new PR.

On a side note, we are currently working on a separate Dakara feeder, which will ease the feeding process.

File: docker/Dockerfile
Comment (pjeanjean) [reply docker/Dockerfile]: Noted, I don't mind waiting for the next release before merging this if you want.

File: docker/Dockerfile
Comment (pjeanjean) [reply docker/Dockerfile]: This tip is very welcome :o

File: docker/entrypoint.sh
Comment (pjeanjean) [reply docker/entrypoint.sh:10]: I honestly never noticed there was a command for this (I have always created the player from the web interface...)

File: docker/Dockerfile
Comment (pjeanjean) [reply docker/Dockerfile]: I wanted to do this at first, but I couldn't find an elegant way to do this and still keep the default behaviour...

**Comment by pjeanjean** [comment]
Should be good now.

Note: Right now, I am using the development settings profile.
I will check later how I can use the production profile properly.

**Comment by Neraste** [comment]
For a local use, the development config is fine. The production config is designed for serving the application on a real server.

File: docker/Dockerfile
Comment (Neraste) [inline docker/Dockerfile]: Just for consistency, you should use the absolute path to the requirement file: `/dakaraneko/karaneko/requirements.txt`.

**Comment by Neraste** [comment]
As a side note, you should **never** force push to a repository. It's a good rule to follow.

**Review by Neraste — COMMENTED** [review]
Sorry for late code review. Everything is in place, there is just one minor typo change and it will be perfect.

File: docker/Dockerfile
Comment (pjeanjean) [reply docker/Dockerfile]: Also sorry for the wait, it's done~~

**Review by Neraste — APPROVED** [review]

**Comment by Neraste** [comment]
OK, this is perfect. Thanks for the PR.

**Pull request merged**
